src/main/resources/changelog/changes/v1.0.0/create-kb-credentials-table.xml

```diff
@@ -0,0 +1,88 @@
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+    http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd">
+
+  <changeSet id="task-001@@create-kb-credentials-table" author="sherzod">
+    <createTable tableName="kb_credentials">
+      <column name="id" type="uuid">
+        <constraints primaryKey="true" primaryKeyName="pk_kb_credentials"
+                     nullable="false"/>
+      </column>
+      <column name="name" type="varchar(255)">
+        <constraints unique="true" uniqueConstraintName="unq_kb_credentials_name"
+                     nullable="false"/>
+      </column>
+      <column name="customer_id" type="varchar(100)">
+        <constraints nullable="false"/>
+      </column>
+      <column name="api_key" type="varchar(100)">
+        <constraints nullable="false"/>
+      </column>
+      <column name="url" type="varchar(255)">
+        <constraints nullable="false"/>
+      </column>
+      <column name="created_date" type="timestamptz" defaultValueComputed="current_timestamp">
+        <constraints nullable="false"/>
+      </column>
+      <column name="created_by_user_id" type="uuid">
+        <constraints nullable="false"/>
+      </column>
+      <column name="created_by_user_name" type="varchar(100)">
+        <constraints nullable="false"/>
+      </column>
```

Comment:
Probably usernames could be removed from table.

Reply:
removed